Installation/cmake/modules/CGAL_Eigen_support.cmake

```diff
@@ -0,0 +1,9 @@
+if(EIGEN3_FOUND AND NOT TARGET CGAL::Eigen_support)
+  if(NOT TARGET Threads::Threads)
+    find_package(Threads REQUIRED)
+  endif()
```

Comment:
@sgiraudot If you are still around, do you remember why you search for CMake `Threads` support in a file about Eigen3? Was that a copy-paste error?